text/0000-default-scoped-light.md

```diff
@@ -0,0 +1,132 @@
+
+```js
+import _implicitStylesheets from './component.css?scoped=true';
+import _implicitScopedStylesheets from './component.scoped.css?scoped=true';
+import _implicitUnscopedStylesheets from './component.scoped.css';
+```
+
+...whereas in shadow DOM, it would be:
+
+```js
+import _implicitStylesheets from './component.css';
+import _implicitScopedStylesheets from './component.scoped.css?scoped=true';
+import _implicitUnscopedStylesheets from './component.scoped.css';
+```
+
+</details>
+
+## Drawbacks
+
+### Breaking existing behavior
+
+Light DOM is considered "beta" and has not officially shipped in LEX yet. However, some folks may be using it off-core. In those cases, the behavior will change, and they will have to fix their existing components, if they were relying on `*.css` being unscoped for light DOM components.
+
+### More complexity in the design
+
+The old system (`*.css` is unscoped, `*.scoped.css` is scoped) was simple to describe. The new system has a 3x3 table to explain how it works.
+
+Arguably the new system is "simpler," in that developers can mostly just use `*.css` and it will be "scoped." However, once you get into the weeds of what "scoped" actually means (e.g. the fact that scoping is useful for shadow DOM as well, e.g. in the case of a shadow-parent-light-child situation or [multiple templates](https://lwc.dev/guide/html_templates#render-multiple-templates)), then the existing system is actually much simpler.
+
+### Breaking changes for bundlers/adapters
+
+We will need to change the existing behavior in the LWC adapters for Rollup, Jest, Webpack, LWR, lwc-platform, and anywhere else that implements the LWC resolution algorithm. We may break existing unit tests and apps while getting the feature out the door. 
+
+## Alternatives
+
+### Make scoped styles the default everywhere
+
+We cannot do this, because it would potentially break existing shadow DOM components. Even forgetting that light DOM exists, the addition of an extra class selector could change the specificity of the CSS rules, and thus the runtime styling behavior (in both synthetic and native).
+
+### Keep the existing system
+
+We could _not_ do this proposal, and teach developers how to use scoped styles in the existing system.
+
+Arguably, this approach is not terrible because:
+
+1) Developers already have a lot of new concepts to learn with light DOM (`lwc:render-mode`, `renderMode`, what `:host` means, security, etc.) – this is just one extra concept to learn.
+2) 99% of the time, developers can just remember "Always use `*.scoped.css`," and it will match their intuition. For instance, even in the case of native shadow DOM, using scoped styles doesn't add a huge performance tax, and it avoids the aforementioned edge cases with shadow-parent-light-child and multi-template components.
+
+## Adoption strategy
+
+It will be a breaking change in some cases. We will have to communicate it very clearly.

```

Comment:
Since the API is already beta on the Salesforce platform, all the changes have to be backward compatible. This change will need to be behind a compiler flag to avoid potentially breaking existing light DOM components on the platform.